backend/src/sockets/index.js

```diff
@@ -1,3 +1,4 @@
 export {joinRoom} from './joinRoom';
 export {disconnect} from './disconnect';
 export {start} from './start';
+export {setPrefences} from '././setPreferences';
```

Comment:
spelling typo

Reply:
should be setPreferences :)

Reply:
also the import path is incorrect, should be './setPreferences' I believe

Reply:
oops dunno how that second one happened, should be fixed now